homeassistant/components/timer/__init__.py

```diff
@@ -220,9 +258,14 @@ def state(self):
     def state_attributes(self):
         """Return the state attributes."""
         return {
-            ATTR_DURATION: str(self._config[CONF_DURATION]),
+            ATTR_DURATION: _time_str(self._config[CONF_DURATION]),
             ATTR_EDITABLE: self.editable,
-            ATTR_REMAINING: str(self._remaining),
+            ATTR_REMAINING: _time_str(self._remaining),
+            ATTR_RESTORE: str(self._restore),
+            ATTR_RESTORE_GRACE_PERIOD: str(self._restore_grace_period),
+            ATTR_END: str(self._end.replace(tzinfo=timezone.utc).astimezone(tz=None)) \
```

Comment:
Please limit the PR only to the changes that are in the description. Remove this.

Reply:
You're referring to the timer > 24 hours issue right? Using `_time_str()` here instead of `str()`, use a separate pull request and come up with a better solution? So I should revert this back to `str()` for now and issue a new PR for allowing for longer timers?